src/utils/file.ts

```diff
@@ -32,20 +32,41 @@ export const getDirectories = async (source: string) => {
   );
 };
 
-export const getFiles = async (dirPath: string, pattern: RegExp) => {
-  const dirs = await readdir(dirPath, { withFileTypes: true });
-
-  return (
-    await filterAsync(dirs, async (f: Dirent | string) => {
-      try {
-        if (typeof f === 'string') {
-          return (await exists(path.join(dirPath, f))) && pattern.test(f);
-        } else {
-          return f.isFile() && pattern.test(f.name);
+export const getFiles = async (
```

Comment:
I had to refactor this function because filtering wouldn't allow expanding deep folders.
Feel free to suggest a different approach.